src/main/java/kitchenpos/menu/ui/MenuRestController.java

```diff
@@ -0,0 +1,39 @@
+package kitchenpos.menu.ui;
+
+import java.net.URI;
+import java.util.List;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import kitchenpos.menu.application.MenuService;
+import kitchenpos.menu.dto.MenuRequest;
+import kitchenpos.menu.dto.MenuResponse;
+
+@RestController
+public class MenuRestController {
+	private final MenuService menuService;
+
+	public MenuRestController(final MenuService menuService) {
+		this.menuService = menuService;
+	}
+
+	@PostMapping("/api/menus")
+	public ResponseEntity<MenuResponse> create(@RequestBody final MenuRequest menuRequest) {
+		final MenuResponse created = menuService.create(menuRequest);
+		final URI uri = URI.create("/api/menus/" + created.getId());
+		return ResponseEntity.created(uri)
```

Comment:
이 부분도 자주 반복되니 아예 ResponseEntity를 반환하는 CommonResponse 같은 객체가 있으면 어떨까요?? 🤔 
```suggestion
		return CommonResponse.create(uri, body);
```
이 부분도 참고만 부탁 드리겠습니다 😄